docs/asf.md

```diff
@@ -0,0 +1,888 @@
+the process.
+
+PouchDB:
+PouchDB makes significant use of GitHub Actions for CI. For 2024, we have used
+214,556 total minutes across 53,831 job runs. This can be migrated if need be
+but it’d be great if this carefully crafted setup could remain in place.
+
+### Initial Committers
+
+List of committers (stating name and an email address) used to bootstrap the
+community. Mark each which has submitted a contributor license agreement (CLA).
+Existing committers should use their apache.org email address (since they
+require only appropriate karma). Others should use the email address that is (or
+will be) on the CLA. That makes it easy to match CLAs with proposed committers
+to the project.
+
+It is a good idea to submit CLAs at the same time as the proposal. Nothing is
+lost by having a CLA on file at Apache but processing may take some time.
+
+Note this and this. Consider creating a separate section where interested
+developers can express an interest (and possibly leave a brief introduction) or
+ask them to post to the general list.
+
+Example (OpenJPA):
+> Abe White (awhite at bea dot com)
+> Marc Prud'hommeaux (mprudhom at bea dot com)
+> Patrick Linskey (plinskey at bea dot com)
+> ...
+> Geir Magnusson Jr (geirm at apache dot org) *
+> Craig Russell (clr at apache dot org) *
+
+PouchDB:
+
+Alba Herrerías Ramírez albaherreriasdev at gmail dot com
+Alex Anderson alexanderandersonofandover at gmail dot com
+Diana Thayer garbados at apache dot org (CLA)
+Diana Barsan twisteddiana at gmail dot com
+Gareth Bowen gareth at bowenwebdesign dot co dot nz
+Jan Lehnardt jan at apache dot org (CLA)
+James Coglan james at neighbourhood dot ie
+Johannes Schmidt schmidt at tf-fabrik dot de
+Steven-John Lange sourcer85 at gmail dot com
+
+
+### Sponsors
+
+A little bit of a controversial subject. Committers at Apache are individuals
+and work here on their own behalf. They are judged on their merits, not their
+affiliations. However, in the spirit of full disclosure, it is useful for any
+current affiliations which may affect the perceived independence of the initial
+committers to be listed openly at the start.
+
+For example, those in salaried positions whose job is to work on the project
+should list their affiliation. Having this list helps to judge how much
+diversity exists in the initial list and so how much work there is to do.
+
+This is best done in a separate section away from the committers list.
+
+Only the affiliations of committers on the initial bootstrap list are relevant.
+These committers have not been added by the usual meritocratic process. It is
+strongly recommended that once a project is bootstrapped, developers are judged
+by their contributions and not by their background. This list should not be
+maintained after the bootstrap has been completed.
+
+PouchDB:
+Alba Herrerías Ramírez Neighbourhoodie
+Alex Anderson independent, formerly Medic
+Diana Thayer independent
```

Comment:
I prefer to go by Diana Belle or Diana Fernanda Belle, unless this is a matter of legal documentation.